kpi/serializers/v2/user.py

```diff
@@ -29,12 +29,12 @@ class Meta:
         fields = ('url',
                   'username',
                   'assets',
-                  'member_since',
+                  'date_joined',
                   'public_collection_subscribers_count',
                   'public_collections_count',
                   )
 
-    def get_member_since(self, user):
+    def get_date_joined(self, user):
```

Comment:
This is already done magically by Django because `date_joined` is already a property of the model `User`. 

As I said above, we can keep it to override the output. In the `CurrentUserSerializer`, `date_joined` is overridden. We could do the same over here for consistency.
What do you think? 

https://github.com/kobotoolbox/kpi/blob/2795-change-member-since-to-date-joined/kpi/serializers/current_user.py#L54-L56

Reply:
@noliveleger I think consistency is nice across the different serializers, so perhaps we either change them all to allow Django to handle them or continue with the current "convention"?